code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -21,13 +21,25 @@
 	family_heirlooms = list(/obj/item/toy/cattoy)
 	examine_limb_id = SPECIES_HUMAN
 
-// Prevents felinids from taking toxin damage from carpotoxin
+// Prevents felinids from taking toxin damage from carpotoxin and makes them throw up because of chocolate.
 /datum/species/human/felinid/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H, seconds_per_tick, times_fired)
 	. = ..()
 	if(istype(chem, /datum/reagent/toxin/carpotoxin))
 		var/datum/reagent/toxin/carpotoxin/fish = chem
 		fish.toxpwr = 0
-
+	if(istype(chem, /datum/reagent/consumable/coco) || istype(chem, /datum/reagent/consumable/hot_coco) || istype(chem, /datum/reagent/consumable/milk/chocolate_milk) || istype(chem, /datum/reagent/consumable/chocolatepudding))
```

Comment:
well, this i hate. anyone adds a chocolate drink, reagent, anything and they have to remember to update this increasingly large list of chocolate types

Reply:
the ideal way to fix it is to add a new food category: chocotastic

Reply:
So they puke because of chocolate cake which doesn't have any cocoa?